src/ActinomycetacaeBotManager.ts

```diff
@@ -1,6 +1,8 @@
 import * as config from "../../config.json";
 import {CommandoClient} from "discord.js-commando";
 import {BotManager, GuildAudioPlayer, SoundFileManager} from "discord-shine";
+import { RolesManager } from "./utility/RolesManager.js";

```

Comment:
More of a comment and not suggesting a change, we definitely want to move to this sort of spacing in the future instead of the 0 space after brackets shown in other import statements.